Common/UI/Screen.cpp

```diff
@@ -149,8 +149,8 @@ void ScreenManager::resized() {
 	std::lock_guard<std::recursive_mutex> guard(inputLock_);
 	// Have to notify the whole stack, otherwise there will be problems when going back
 	// to non-top screens.
-	for (auto iter = stack_.begin(); iter != stack_.end(); ++iter) {
-		iter->screen->resized();
+	for (auto &layer : stack_) {
+        layer.screen->resized();
```

Comment:
tabs

Reply:
fix it https://github.com/hrydgard/ppsspp/pull/18559/commits/37ce2a4de179e6931b8f7e5de4c78728ac0c5c5b